src/common/lib/theme/helpers.ts

```diff
@@ -0,0 +1,15 @@
+export const BORDER_STYLES = [
+  { name: "Theme Border", value: "var(--user-theme-border-width)" },
```

Comment:
should this be `var(--user-theme-fidget-border-width)`

Reply:
oops, my bad. updated